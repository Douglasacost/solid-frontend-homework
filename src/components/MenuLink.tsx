type Props = {
  text: string,
  icon: string,
  logout?: boolean,
}

const MenuLink = ({ icon, text, logout }: Props) => (
  <div className={`dropdown__section_item ${logout ? 'dropdown__section_item_logout' : ''}`}>
    <i className="material-icons-outlined">
      {icon}
    </i>

    {text}
  </div>
)

export default MenuLink
