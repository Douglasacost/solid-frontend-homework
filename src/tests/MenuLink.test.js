import { shallow } from 'enzyme'
import MenuLink from '../components/MenuLink'

describe('<MenuLink />', () => {
  it('renders link', () => {
    expect(shallow(<MenuLink icon="phone" text="Call me" />)).toMatchSnapshot()
  })
})
