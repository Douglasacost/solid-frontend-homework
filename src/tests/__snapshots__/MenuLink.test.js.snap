// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MenuLink /> renders link 1`] = `
<div
  className="dropdown__section_item "
  role="button"
>
  <i
    className="material-icons-outlined"
  >
    phone
  </i>
  Call me
</div>
`;
